.gitignore

```diff
@@ -0,0 +1,17 @@
+
+.vscode/c_cpp_properties.json
+.vscode/launch.json
+.vscode/settings.json
+.vscode/settings.json
+edweiss-app/package.json
+edweiss-app/package-lock.json
+edweiss-app/package-lock.json
+edweiss-app/app.config.ts
+edweiss-app/package-lock.json
+edweiss-app/package-lock.json
+.vscode/settings.json
+.vscode/settings.json
+.vscode/settings.json
+edweiss-app/package-lock.json
+.vscode/settings.json
+edweiss-app/package.json
```

Comment:
Hey, thanks for the question! Regarding the package-lock.json, I understand your concern. The reason I had it in the .gitignore was to avoid any potential conflicts with different package versions across environments. But I agree with you—generally, it's a good practice to commit it to ensure consistent dependencies, so I'll remove it from the .gitignore.

Reply:
Thanks happy to feel convinced by ma argument.